[31m[ERROR] line 10: Number of arguments for 'a': 2, 3 given[0m
